### PR TITLE
Updated jans version from 19_dev to latest 20-1

### PR DESCRIPTION
**Pull request by corriganjeff**
Updated jans version from 19_dev to latest 20-1

### CODE REVIEW

File: docker-jans-monolith/jans-mysql-compose.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ services:
       - MYSQL_PASSWORD=1t5Fin3#security
       - MYSQL_ROOT_PASSWORD=1t5Fin3#security
   jans:
-    image: ghcr.io/janssenproject/jans/monolith:1.0.19_dev
+    image: ghcr.io/janssenproject/jans/monolith:1.0.20-1
```
Comment (iromli) [inline docker-jans-monolith/jans-mysql-compose.yml]: The monolith image is mostly used for quickstart/testing bleeding-edge version of Janssen. In my opinion, it's better to use `:1.0.20_dev` instead.

Comment (corriganjeff) [reply docker-jans-monolith/jans-mysql-compose.yml]: done.

**Comment by iromli** [comment]
@corriganjeff we have released `:1.0.21_dev` recently. It would be nice if you can use the new version. Also, updating `jans-ldap-compose.yml` and `jans-postgres-compose.yml` would be great.

Thanks.

**Comment by corriganjeff** [comment]
> @corriganjeff we have released `:1.0.21_dev` recently. It would be nice if you can use the new version. Also, updating `jans-ldap-compose.yml` and `jans-postgres-compose.yml` would be great.
> 
> Thanks.

Done.

**Review by iromli — APPROVED** [review]

**Pull request merged**